src/utils/utils.js

```diff
@@ -4,8 +4,19 @@ export const prettyLocaleDate = date =>
     month: 'short',
     year: 'numeric',
   })
+
 export const prettyCurrency = amount =>
   parseFloat(amount).toLocaleString('en-US', {
     style: 'currency',
     currency: 'GBP',
   })
+
+export function getTotalAmount(price, quantity) {
+  return price && quantity ? +price * +quantity : 0
+}
+
+export const convertToNumber = (name, value) => {
+  if (name === 'name') return value
+  if (value !== '') return +value
+  return value
+}
```

Comment:
Bularga avval komment qoldirgandim, hali ham to'g'irlanmabtiku

Reply:
ha bularni oldingi brenchda tugirlagan edim, ozeoq chlakashlik buligan shekilli, hal qilaman